doc/CMakeLists.txt

```diff
@@ -13,11 +13,17 @@ if(DOXYGEN_FOUND)
         add_custom_target(doc)
     endif()
 
+    add_custom_target(cpp-apidoc-content-tags
```

Comment:
add_custom_target allows multiple COMMAND statements so there is no need for an explicit `cpp-apidoc-content-tags` target but instead you can add the execution statement as the first COMMAND to the cpp-apidoc target. 

https://cmake.org/cmake/help/v3.1/command/add_custom_target.html